var/spack/repos/builtin/packages/numactl/package.py

```diff
@@ -16,6 +16,9 @@ class Numactl(AutotoolsPackage):
 
     license("LGPL-2.1-only")
 
+    version("2.0.18", sha256="8cd6c13f3096e9c2293c1d732f56e2aa37a7ada1a98deed3fac7bd6da1aaaaf6")
+    version("2.0.17", sha256="af22829cda8b5bdee3d280e61291697bbd3f9bd372afdf119c9348b88369d40b")
+    version("2.0.16", sha256="a35c3bdb3efab5c65927e0de5703227760b1101f5e27ab741d8f32b3d5f0a44c")
     version("2.0.14", sha256="1ee27abd07ff6ba140aaf9bc6379b37825e54496e01d6f7343330cf1a4487035")
```

Comment:
There are a number of patches that apply to 2.0.14, including on pgi and nvhpc compilers. Did you make sure these are not needed anymore? They might be the cause of the failing build in CI.

Reply:
Ah, looks I missed few items for 2.0.16:

We still need [fix-empty-block.patch](https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/numactl/fix-empty-block.patch) in [2.0.16](https://github.com/numactl/numactl/blob/v2.0.16/configure.ac#L20), but not [2.0.17](https://github.com/numactl/numactl/blob/v2.0.17/configure.ac#L23).

We also need the [configure.ac](https://github.com/numactl/numactl/blob/v2.0.16/configure.ac) change from [link-with-latomic-if-needed.patch](https://github.com/spack/spack/blob/9b077a360e73ddeadf473f99530c082d9cb83685/var/spack/repos/builtin/packages/numactl/link-with-latomic-if-needed.patch) to link with libatomic in [2.0.16](https://github.com/numactl/numactl/blob/v2.0.16/configure.ac), but not [2.0.17](https://github.com/numactl/numactl/blob/v2.0.17/configure.ac#L30).

I'll update the package.py and pull the libatomic link into another patch by itself for 2.0.16.